src/pages/users/UsersPage.tsx

```diff
@@ -159,13 +159,57 @@ const UsersPage: FunctionComponent = () => {
                     headerId: 'users.table.error.delete',
                 })
             )
-            .then(() => gridContext?.refresh?.());
+            .then(() => {
+                // close the dialog
+                setShow(false);
+                gridContext?.refresh?.();
+            });
     }, [gridContext, rowsSelection, snackError]);
     const deleteUsersDisabled = useMemo(
         () => rowsSelection.length <= 0,
         [rowsSelection.length]
     );
 
+    const handleDeletion = (value: boolean) => {
+        setShow(value);
+    };
+
+    const buildTitle = useCallback(
+        (users: UserInfos[]) => {
+            const hasMultipleItems = users.length > 1;
+            const descriptor = {
+                id: hasMultipleItems
+                    ? 'users.form.delete.multipleItemsDialogTitle'
+                    : 'users.form.delete.title',
```

Comment:
these 2 keys names should be more consistent. I made a suggestion below.

Reply:
Done